src/components/base/materialcard/templates/templates.js

```diff
@@ -61,6 +61,73 @@ export function templateForVerticalWorkCard(material) {
   };
 }
 
+/**Used in Series page */
+export function templateForBigWorkCard(material) {
+  const fullTitle = material?.titles?.full?.join(": ");
+  const creators = material?.creators;
+  const abstract = material?.abstract;
+
+  const coverSrc = getCoverImage(material.manifestations.mostRelevant);
+
+  const readThisFirst = material?.series?.[0]?.readThisFirst;
+  const numberInSeries = material?.series?.[0]?.numberInSeries?.number;
```

Comment:
bare til advarsel så tænke jeg at deprecate `number` - men det kan vi lige snakke om hvis det er.

Reply:
Jeg tror vi bliver nødt til at beholde den, da display er mangelfuld og/eller formateret uhensigtsmæssigt. Display har sjældent alle dele af serienummeret med (f.eks. kan du se på serien for "Harry Potter (bøger)")